README.md

```diff
@@ -1,6 +1,36 @@
 # transmogrifier
 
-Data transformation CLI app for TIMDEX
+An application to transform source records to the TIMDEX data model to facilitate ingest into an OpenSearch index.
+
+## Description
+
+TIMDEX ingests records from various sources with different metadata formats, necessitating an application to transform those source records to a common metadata format. This application processes both XML and JSON source records and outputs a JSON file of records formatted according to the TIMDEX data model. 
```

Comment:
I don't know how to word it, but I think it could be helpful to draw a straight line between "common metadata format" and "TIMDEX data model", that they are largely one and the same.

Something you're probably wise not to introduce here is the "contract" between Transmog (python class that codifies the data model) and TIM (Opensearch mapping JSON that codifies that data model).  I would argue the data model is the combination and relationship of those two things, but unsure if that complexity belongs here.

Reply:
Good call, will do!